spec/models/ontology_spec.rb

```diff
@@ -339,6 +339,26 @@
         expect(ontology.sentences_count).to eq(ontology.sentences.count)
       end
     end
+
+    context 'axioms count' do
+      it 'should be correct' do
+        expect(ontology.axioms.count).to eq(1)
+      end
+
+      it 'should be reflected in the corresponding field' do
+        expect(ontology.axioms_count).to eq(ontology.axioms.count)
```

Comment:
why do you test something to be equal to itself? What does this test really test? non-nil?